.github/workflows/validation.yaml

```diff
@@ -0,0 +1,52 @@
+name: Validation
+on:
+  push:
+    branches:
+      - '*'
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref || github.run_id }}
+  cancel-in-progress: true
+
+env:
+  NODE_OPTIONS: '--max-old-space-size=6144'
+  NODE_ENV: test
+
+jobs:
+  validate_branch_name:
+    name: Validate branch name
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Validate branch name
+        run: packages/devops/scripts/ci/validateBranchName.sh
+
+  validate_migration:
+    name: Validate new migrations
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Validate new migration
+        run: packages/devops/scripts/ci/validateNewMigrations.sh
+
+  #    - TODO: use eslint no-exclusive-tests flag at lint step after RN-201
+  validate_tests:
+    name: Validate unit tests
+    strategy:
+      matrix:
+        node:
+          - '14.19.3'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
```

Comment:
Oh this sets up node? Cool 😎

Reply:
Yeah GHA use this kind of format to call actions. You can create a new action or call a third party action from marketplace too.